v1.0/v1.0/parameter-reference.cwl

```diff
@@ -0,0 +1,14 @@
+cwlVersion: v1.0
+class: CommandLineTool
+baseCommand: "true"
```

Comment:
This test would be stronger if the correct behavior was communicated through the outputs, not merely through "not failing" :-) How about making the `baseCommand: echo` or use `args.py` like in https://github.com/common-workflow-language/common-workflow-language/blob/master/v1.0/v1.0/inline-js.cwl#L11 ?